--- /home/stellarmate/PiFinder_Stellarmate/.gemini/tmp/index.tpl.orig	2025-11-10 21:19:37.004793790 +0100
+++ /home/stellarmate/PiFinder_Stellarmate/.gemini/tmp/index.tpl	2025-11-10 21:14:26.170342805 +0100
@@ -11,7 +11,7 @@
       <i class="material-icons medium">wifi</i>
     </td>
     <td class="grey-text text-lighten-1">{{wifi_mode}} Mode<br>{{network_name}}<br>{{ip}}</td>
-    <td><a href="/network" class="grey-text"><i class="material-icons">edit</i></a></td>
+    <td></td>
 </tr>
 <tr>
     <td>
