--- /home/stellarmate/PiFinder_Stellarmate/src_pifinder/python/PiFinder/main.py.orig	2025-11-09 17:23:37.363553945 +0100
+++ /home/stellarmate/PiFinder_Stellarmate/src_pifinder/python/PiFinder/main.py	2025-11-09 17:23:37.351554169 +0100
@@ -919,6 +919,8 @@
         gps_type = cfg.get_option("gps_type")
         if gps_type == "ublox":
             gps_monitor = importlib.import_module("PiFinder.gps_ubx")
+        elif gps_type == "stellarmate":
+            gps_monitor = importlib.import_module("PiFinder.gps_stellarmate")
         else:
             gps_monitor = importlib.import_module("PiFinder.gps_gpsd")
 
