--- /tmp/menu_structure.py.pristine_original	2025-11-09 19:04:25.605620135 +0100
+++ /home/stellarmate/PiFinder_BAK/python/PiFinder/ui/menu_structure.py	2025-11-09 11:24:30.058881889 +0100
@@ -963,6 +963,10 @@
                             "name": _("GPSD (generic)"),
                             "value": "gpsd",
                         },
+                        {
+                            "name": _("Stellarmate"),
+                            "value": "stellarmate",
+                        },
                     ],
                 },
                 {
