--- python/PiFinder/ui/menu_structure.py.orig	2026-05-31 17:40:06.308609391 +0200
+++ python/PiFinder/ui/menu_structure.py	2026-05-31 17:39:02.147942040 +0200
@@ -897,24 +897,7 @@
                         },
                     ],
                 },
-                {
-                    "name": _("WiFi Mode"),
-                    "class": UITextMenu,
-                    "select": "single",
-                    "value_callback": callbacks.get_wifi_mode,
-                    "items": [
-                        {
-                            "name": _("Client Mode"),
-                            "value": "Client",
-                            "callback": callbacks.go_wifi_cli,
-                        },
-                        {
-                            "name": _("AP Mode"),
-                            "value": "AP",
-                            "callback": callbacks.go_wifi_ap,
-                        },
-                    ],
-                },
+
                 {
                     "name": _("Mount Type"),
                     "class": UITextMenu,
@@ -1015,6 +998,10 @@
                                             "name": _("GPSD (generic)"),
                                             "value": "gpsd",
                                         },
+                                        {
+                                            "name": _("Stellarmate"),
+                                            "value": "stellarmate",
+                                        },
                                     ],
                                 },
                                 {
