--- python/PiFinder/ui/menu_structure.py.orig
+++ python/PiFinder/ui/menu_structure.py
@@ -994,24 +994,7 @@ pifinder_menu = {
                         },
                     ],
                 },
-                {
-                    "name": _("WiFi Mode"),
-                    "class": UITextMenu,
-                    "select": "single",
-                    "value_callback": callbacks.get_wifi_mode,
-                    "items": [
-                        {
-                            "name": _("Client Mode"),
-                            "value": "Client",
-                            "callback": callbacks.go_wifi_cli,
-                        },
-                        {
-                            "name": _("AP Mode"),
-                            "value": "AP",
-                            "callback": callbacks.go_wifi_ap,
-                        },
-                    ],
-                },
+
                 {
                     "name": _("Mount Type"),
                     "class": UITextMenu,
@@ -1112,6 +1095,10 @@ pifinder_menu = {
                                             "name": _("GPSD (generic)"),
                                             "value": "gpsd",
                                         },
+                                        {
+                                            "name": _("Stellarmate"),
+                                            "value": "stellarmate",
+                                        },
                                     ],
                                 },
                                 {
@@ -1217,7 +1204,6 @@ pifinder_menu = {
                     ],
                 },
                 {"name": _("Console"), "class": UIConsole},
-                {"name": _("Software Upd"), "class": UISoftware},
                 {"name": _("Test Mode"), "callback": callbacks.activate_debug},
                 {
                     "name": _("Experimental"),
